### ISSUE TITLE
Header Disappears

### ISSUE THREAD

**Issue by sbryfcz** [shprink/ionic-native-transitions#68]
So I just found this plugin/ionic wrapper and its amazing. Expect for 1 little thing. When I tap a item to transition to the next page, the current page's header will sometimes disappear. It will go blank as the next view slides over it. I'm having trouble really tracking down the issue. Very odd since it seems random. Has anyone ever run into this issue? Keep up the great work!

**Comment by jeremygomez** [comment]
+1 
I just noticed this in my testing as well. It would be nice if there were a way to combine the native transition with Ionic's (much smoother) navbar transition. That way the header and back button would slide and fade nicely while the main view transition is handled by the native transition plugin.

**Comment by shprink** [comment]
You can try that using the `fixedPixelsTop` option.

**Comment by jeremygomez** [comment]
I am using a combination of `$ionicNativeTransitions.enable(true, false)` and `fixedPixelsTop`, and although I am not seeing the "disappearing header" that this issue originally states (and that I was seeing), for some reason the navbar titles and back buttons are not animating during transition. I am just seeing them show/hide when the native transition completes.

**Comment by shprink** [comment]
According to this code, the second arugment set to false will enable the ionic transitions:

https://github.com/shprink/ionic-native-transitions/blob/develop/lib/provider.js#L244-L250

Maybe a bug? Let me know if you find a fix

**Comment by jeremygomez** [comment]
Yeah, I was hoping that by enabling Ionic that the navbar transition would work by default. There doesn't appear to be anything wrong in the `enableFromService()` code. Perhaps something is amiss with trying to transition the navbar without the associated pane. Either way, I have posted a couple of videos to illustrate what I am seeing. I'll keep digging to see if I can figure anything out.

Stock Ionic transition (`$ionicNativeTransitions.enable(false)`)
https://www.dropbox.com/s/30acql2d8bpdeca/native-transition-header-ionic.mov?dl=0

Combo native/Ionic transition (`$ionicNativeTransitions.enable(true, false)`)
https://www.dropbox.com/s/5q63fwrt6afzo72/native-transition-header-plugin.mov?dl=0

**Comment by MrPro100** [comment]
+1
